configure.ac

```diff
@@ -1,7 +1,7 @@
 #                                               -*- Autoconf -*-
 # Process this file with autoconf to produce a configure script.
 AC_PREREQ([2.69])
-AC_INIT([opx-base-model], [3.151.0], [ops-dev@lists.openswitch.net])
+AC_INIT([opx-base-model], [3.152.0], [ops-dev@lists.openswitch.net])
```

Comment:
package version in debian/changelog and confugure.ac should be same : '3.151.0+num1'

Reply:
Version should be 3.151.0+opx1